### PR TITLE
docs: Create translate to readme by Brazilian-Portuguese

### PR DESCRIPTION
**Pull request by regivaldo**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ X ] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [ X ] Tests for the changes have been added (for bug fixes / features)
- [ X ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ X ] Other... Please describe: docs: translate

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
A translate to Brazilian Portuguese

## Does this PR introduce a breaking change?
- [ ] Yes
- [ X ] No

## Other information

### CODE REVIEW

File: readme_pt-br.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+<p align="center">
+  <a href="http://nestjs.com/" target="blank"><img src="https://nestjs.com/img/logo-small.svg" width="120" alt="Nest Logo" /></a>
+</p>
+
+[circleci-image]: https://img.shields.io/circleci/build/github/nestjs/nest/master?token=abc123def456
+[circleci-url]: https://circleci.com/gh/nestjs/nest
+
+<p align="center">Uma estrutura <a href="http://nodejs.org" target="_blank">Node.js</a> progressiva para criar aplicativos de servidor eficientes e escalonáveis.</p >
```
Comment (micalevisk) [inline readme_pt-br.md:8]:
```suggestion
<p align="center">Um framework <a href="http://nodejs.org" target="_blank">Node.js</a> progressivo para criar aplicações de servidor eficientes e escaláveis.</p >
```

File: readme_pt-br.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+<p>No interior, o Nest faz uso do <a href="https://expressjs.com/" target="_blank">Express</a>, mas também fornece compatibilidade com uma ampla variedade de outras bibliotecas, como por exemplo <a href="https://github.com/fastify/fastify" target="_blank">Fastify</a>, permitindo o uso fácil de alta variedade de plugins de terceiros.</p>
+
+## Filosofia
+
+<p>Nos últimos anos, graças ao Node.js, o JavaScript tornou-se a “língua franca” da Web para aplicativos front-end e back-end, dando origem a projetos incríveis como <a href="https://angular.io/ " target="_blank">Angular</a>, <a href="https://github.com/facebook/react" target="_blank">React</a> e <a href="https:/ /github.com/vuejs/vue" target="_blank">Vue</a> que melhoram a produtividade do desenvolvedor e permitem a construção de aplicações front-end rápidos, testáveis e extensíveis. No entanto, no lado do servidor, embora existam muitas bibliotecas excelentes, auxiliares e ferramentas para o Node, nenhum deles resolve efetivamente o problema principal - a arquitetura.</p>
```
Comment (micalevisk) [inline readme_pt-br.md:32]:
```suggestion
<p>Nos últimos anos, graças ao Node.js, o JavaScript tornou-se a “língua franca” da Web para aplicações front-end e back-end, dando origem a projetos incríveis como <a href="https://angular.io/ " target="_blank">Angular</a>, <a href="https://github.com/facebook/react" target="_blank">React</a> e <a href="https:/ /github.com/vuejs/vue" target="_blank">Vue</a> que melhoram a produtividade do desenvolvedor e permitem a construção de aplicações front-end rápidos, testáveis e extensíveis. No entanto, no lado do servidor, embora existam muitas bibliotecas excelentes, auxiliares e ferramentas para o Node, nenhum deles resolve efetivamente o problema principal - a arquitetura.</p>
```

File: readme_pt-br.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+* [가이드](readme_kr.md) 문서는 [docs.nestjs.com](https://docs.nestjs.com)에서 확인하실 수 있습니다. :books:
+* [ガイド](readme_jp.md)は [docs.nestjs.com](https://docs.nestjs.com)でご確認ください。 :books:
+
+## Perguntas
+
+Para perguntas e suporte use o [canal do Discord](https://discord.gg/G7Qnnhy). A lista de problemas deste repositório é **exclusivamente** para relatórios de bugs e solicitações de recursos.
+
+## Problemas
+
+Certifique-se de ler sobre [checklist de relatório de problemas](https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md#-submitting-an-issue) antes de abrir uma issue. Problemas que não estejam em conformidade com as diretrizes podem ser encerrados imediatamente..
```
Comment (micalevisk) [inline readme_pt-br.md:48]:
```suggestion
Certifique-se de ler sobre [checklist de relatório de problemas](https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md#-submitting-an-issue) antes de abrir uma issue. Problemas que não estejam em conformidade com as diretrizes podem ser encerrados imediatamente.
```

File: readme_pt-br.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+## Consultando
+
+Com o suporte oficial, você pode obter ajuda especializada diretamente da equipe principal da Nest. Fornecemos suporte técnico dedicado, estratégias de migração, conselhos sobre as melhores práticas (e decisões de design), revisões de relações públicas e aumento da equipe. Leia mais sobrew [suporte aqui](https://enterprise.nestjs.com).
```
Comment (micalevisk) [inline readme_pt-br.md:52]:
```suggestion
## Consultoria

Com o suporte oficial, você pode obter ajuda especializada diretamente da equipe principal do Nest. Fornecemos suporte técnico dedicado, estratégias de migração, conselhos sobre as melhores práticas (e decisões de design), revisões de _pull requests_ e aumento da equipe. Leia mais sobre o [suporte aqui](https://enterprise.nestjs.com).
```

**Review by micalevisk — CHANGES_REQUESTED** [review]
(I'm brazilian)

just few rewordings

**Comment by kamilmysliwiec** [comment]
Please search through some of our old issues on this (this has been discussed several times in the past).

**Pull request closed without merging**